Unable to install coursier

Hi,

First of all, thanks for building this GitHub action, this is great!

I've been having problems when running it though. I'm literally copying and pasting the example from https://github.com/scala-steward-org/scala-steward-action#usage and the action is failing with: `Unable to install coursier`

Here is the output of the step:

```
Launch scala-steward-org/scala-steward-action@v2
  with:
    github-token: ***
    scala-steward-version: 0.5.0-385-e5e4789c-SNAPSHOT
    ignore-opts-files: true
    sign-commits: false
✓ Connected to Maven Central
##[error] ✕ Unable to install coursier
```

Also tried fixing the version to `v2.1.0` and `v2.0.0` and it also fails.

Is anyone else experiencing the same?

Of course, here it is the output:

```
##[debug]Evaluating condition for step: 'Launch Scala Steward'
##[debug]Evaluating: success()
##[debug]Evaluating success:
##[debug]=> true
##[debug]Result: true
##[debug]Starting: Launch Scala Steward
##[debug]Loading inputs
##[debug]Evaluating: secrets.ADMIN_GITHUB_TOKEN
##[debug]Evaluating Index:
##[debug]..Evaluating secrets:
##[debug]..=> Object
##[debug]..Evaluating String:
##[debug]..=> 'ADMIN_GITHUB_TOKEN'
##[debug]=> '***'
##[debug]Result: '***'
##[debug]Loading env
Run scala-steward-org/scala-steward-action@v2
✓ Connected to Maven Central
##[debug]Downloading https://git.io/coursier-cli-linux
##[debug]Destination /home/runner/work/_temp/bbe1660a-10e9-49e6-bafc-38e02ca81513
##[debug]download complete
##[debug]EXDEV: cross-device link not permitted, rename '/home/runner/work/_temp/bbe1660a-10e9-49e6-bafc-38e02ca81513' -> '/home/runner/bin/cs'
##[error] ✕ Unable to install coursier
##[debug]Node Action run completed with exit code 1
##[debug]Finishing: Launch Scala Steward
```

Thanks @alejandrohdezma 

I've just tried the new version, it is still failing unfortunately:

```
Run scala-steward-org/scala-steward-action@07b63bf19f36bd08a077967ce592e872ee9ea171
✓ Connected to Maven Central
##[debug]Downloading https://git.io/coursier-cli-linux
##[debug]Destination /home/runner/work/_temp/4e4e32f6-1342-4c09-9d4c-f8c7ff7b075e
##[debug]download complete
##[debug]EXDEV: cross-device link not permitted, rename '/home/runner/work/_temp/4e4e32f6-1342-4c09-9d4c-f8c7ff7b075e' -> '/home/runner/bin/cs'
##[error] ✕ Unable to install coursier
```

--- a/src/coursier.ts
+++ b/src/coursier.ts
@@ -17,7 +17,8 @@ export async function install(): Promise<void> {
 
     const binPath = '/home/runner/bin'
     await io.mkdirP(binPath)
-    await io.mv(temp, path.join(binPath, 'cs'))
+    await io.cp(temp, path.join(binPath, 'cs'))
+    await io.rmRF(temp)
 
     core.addPath(binPath)
   } catch (error) {

